src/test/ui/attributes/key-value-non-ascii.stderr

```diff
@@ -1,8 +1,8 @@
-error: non-ASCII character in byte constant
+error: byte string literal must be ASCII
```

Comment:
Not sure who's the right reviewer for wording, but `error: <this is what is wrong>` feels better than `error: <this is what is right>`. So perhaps 

```
error: non-ASCII character in byte string literal
```

?

Reply:
I think there is little consistency in existing messages, but I agree your suggestion is an improvement.